src/realm/table_macros.hpp

```diff
@@ -1,936 +0,0 @@
-/*************************************************************************
- *
- * Copyright 2016 Realm Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- **************************************************************************/
-
-#ifndef REALM_TABLE_MACROS_HPP
-#define REALM_TABLE_MACROS_HPP
-
-#include <realm/table_basic.hpp>
```

Comment:
Can't table_basic.hpp be removed too now? As far as I can see, the only reference to it was from here.